rwgame/debugstate.cpp

```diff
@@ -451,18 +451,17 @@ void DebugState::spawnVehicle(unsigned int id)
 void DebugState::spawnFollower(unsigned int id)
 {
 	auto ch = game->getPlayer()->getCharacter();
-	if(! ch) return;
-
-	glm::vec3 fwd = ch->rotation * glm::vec3(0.f, 1.f, 0.f);
+	if (!ch)
+		return;
 
-	glm::vec3 hit, normal;
-	if(game->hitWorldRay(ch->position + (fwd * 10.f), {0.f, 0.f, -2.f}, hit, normal)) {
-		auto spawnPos = hit + normal;
-		auto follower = game->getWorld()->createPedestrian(id, spawnPos);
-		jumpCharacter(game, follower, spawnPos);
-		follower->controller->setGoal(CharacterController::FollowLeader);
-		follower->controller->setTargetCharacter(ch);
-	}
+	auto playerRot = ch->getRotation();
+	auto spawnPos = ch->getPosition();
+	spawnPos += playerRot * glm::vec3(0.f, 3.f, 0.f);
+	auto spawnRot = glm::quat(
+	    glm::vec3(0.f, 0.f, glm::roll(playerRot) + glm::half_pi<float>()));
+	auto follower = game->getWorld()->createPedestrian(id, spawnPos, spawnRot);
```

Comment:
I think the old logic was better here.

It makes sense to rotate a car because large vehicles like the "Coach" are very bad to place properly if it's always facing north. Also it's useful to spawn vehicles in the air/water if the vehicle is used for testing purposes (or a boat later!)

However, a character will fall to his death if not spawned on ground level.
I don't care where a character is facing because AI will take over + it's small enough that rotation doesn't have an impact (especially not with the current capsule collider shape).